cluster-sim/cluster_sim/utils.py

```diff
@@ -88,26 +88,23 @@ class Persisted(object):
     filename = None
     default_state = {}
```

Comment:
If resetting corosync and pacemaker state didn't fix the issue then we did not understand the issue. Resetting more stuff feels like trying things in the hope something works.

The hypothesis is that the fake plugin returns corosync/pacemaker data when they are not virtually started. So stopping them (to simulate uninstall as clear_ha does) and making the code only return when they are started should fix it.

If that does not fix is the hypothesis is wrong and just doing more of the same seems strange to me.